tests/installation/install_and_reboot.pm

```diff
@@ -116,6 +119,8 @@ sub run() {
         # check for right boot-device on s390x (zVM)
         if (check_var('BACKEND', 's390x')) {
             if (script_run('lsreipl | grep 0.0.0150')) {
+                # TODO like above, should we fail because we don't have a
+                # bugref?
                 record_soft_failure 'default bootdevice not set';
```

Comment:
This was introduced as a softfail, because we wanted to get the very first tumbleweed test on s390 passed
still it should stay here, since this is a good test

But yes, I agree - we should fail with a proper error message in the future, instead of just softfailing

I will work on this